hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/data/proxies/ProxySimpleRolloutGroupsDefinition.java

```diff
@@ -20,6 +20,7 @@ public class ProxySimpleRolloutGroupsDefinition implements Serializable {
     private Integer numberOfGroups;
     private String triggerThresholdPercentage;
     private String errorThresholdPercentage;
+    private boolean confirmationRequired = true;
```

Comment:
This "Proxy" entities are reflecting the UI state. Since the confirmation should be required per default, we set this flag here directly "true". If the user consent flow is enabled, this will be read on initializing the Rollout creation dialog and directly checking the corresponding checkbox.  If the consent flow is not enabled. This value will not be considered on Rollout creation.